### DISCUSSION TITLE
another feature request

### DISCUSSION THREAD

**Discussion by Slydylan** [MrHydralisk/Lethal-Company-Enhanced-Radar-Booster#2] [Ideas]
dunno how hard it is to do but it seems like the guys making the helmet camera mods are actively patching this functionality out so
it'd be great if radars would act as security cameras while deployed (showing live feed on the right side monitor & renders all objects and enemies at regular player lod distance)
dunno how but the first 2 versions of solosbodycams have this feature (with the price of horrible lag due to optimizations he hadn't implemented yet) and the camera faces the direction the player was facing when the radar is dropped down. dunno if this is a partially default feature they probably removed it because it was not intended. but its useful :)
It's another cool QOL thing i think would fit in great with what you're going for with this mod. Cheers and happy new year

**Comment by MrHydralisk** [comment]
I used camera in radar booster from helmet camera mod quite a bit, when was playing. It would be quite pity, if they remove that feature. Making a new mod with only such functionality would prob make compatability issues with existing camera mods. So it prob better to persuade helmet camera dev not remove it, rather than making new one.

**Reply by Slydylan** [reply]
also does it render monsters for you too?

**Comment by Slydylan** [comment]
can you tell me what version of helmet cameras you are using because the last time i loaded it up for testing i got no feed on the radar booster

**Reply by MrHydralisk** [reply]
The latest one I think

**Reply by MrHydralisk** [reply]
2.1.5
